filebeat/input/winlog/input.go

```diff
@@ -112,12 +116,18 @@ runLoop:
 			_ = timed.Wait(cancelCtx, 5*time.Second)
 			continue
 		case !api.IsFile() && eventlog.IsChannelNotFound(openErr):
-			log.Errorw("Encountered channel not found error when opening Windows Event Log", "error", openErr)
+			if !channelNotFoundErrDetected {
+				log.Errorw("Encountered channel not found error when opening Windows Event Log", "error", openErr)
+			} else {
+				log.Debugw("Encountered channel not found error when opening Windows Event Log", "error", openErr)
+			}
+			channelNotFoundErrDetected = true
```

Comment:
It's a pity logp doesn't take a level parameter, then this could all be done with that instead of having an auxiliary to manage it.